### ISSUE TITLE
[PNI] Product Headings don't appear on Mobile

### ISSUE THREAD

**Issue by sabrinang** [mozilla/foundation.mozilla.org#4510]
**This is probably an enhancement rather than a bug?**

**To Reproduce**
- go to https://foundation.mozilla.org/en/privacynotincluded/
- view on viewport < 768px

**Expected behavior**
- product name and company headings are seen (similarly to viewports >768px)

**Screenshots**

<img width="450" alt="image" src="https://user-images.githubusercontent.com/2492510/79252572-b51afd00-7e36-11ea-9bcb-ee9e2a6a8090.png">

**Desktop (please complete the following information):**
 - OS: iOS 10.15.4
 - Browser: Firefox 74.0.1 & Chrome 80.0.3987.163

**Smartphone (please complete the following information):**
 - Device: Pixel 2
 - OS: android 10
 - Browse: Firefox 68.7.0 & Chrome 81.0.4044.96

**Additional context**
- This will help from an accessibility perspective too when people do not recognize the look of the product or logo

/cc @natalieworth @caltrider

**Comment by kristinashu** [comment]
Note that we if we add titles to the mobile homepage then we should also add them to the related products at the bottom of product pages.

We had left the titles out on mobile because it gets pretty busy looking. Here's a comp with the company and product title. I think the product title will require an override to reduce the font size and line height. We will also need to reduce the size of the product images.

<img width="318" alt="image" src="https://user-images.githubusercontent.com/1068783/80036262-3cefaf80-84a6-11ea-9ea4-08f45ee8b5e8.png">

cc @mmmavis

**Comment by natalieworth** [comment]
I've mocked up an option (with type overrides needed) [in redpen here](https://redpen.io/p/toa5350e17d1b0efec).

cc @kristinashu @mmmavis @sabrinang @caltrider

**Comment by natalieworth** [comment]
If there are no issues with this, I'm going to close and we can make a dev ticket? Not sure if there's dev capacity yet cc @kristinashu ?

Update to this is in Abstract Sketch document.

![[M] Home Copy 2@2x.jpg](https://images.zenhubusercontent.com/57a10963e40e5714b16cdb3b/4fac7fa2-1e70-4e30-bf92-ec76840fff6e)

**Comment by natalieworth** [comment]
I'm going to close this, but if there are issues or need changes.

Let me know if I should make the dev ticket @kristinashu ? :)

**Comment by kristinashu** [comment]
It looks good to me. @mmmavis are you ok with the overrides? The text really won't fit unless the text is smaller. 

@natalieworth I think we should leave this open until you have the dev ticket, just to make sure we don't forget about it.

**Comment by mmmavis** [comment]
> It looks good to me. @mmmavis are you ok with the overrides? The text really won't fit unless the text is smaller.

Yup yup.

@natalieworth -  just wanna make sure the small grey text on grey background pass accessibility readability & contrast test.

**Comment by natalieworth** [comment]
@mmmavis Yes! The light grey failed for the AAA, so I've changed it to #333333.
<img width="736" alt="Screen Shot 2020-05-15 at 2 38 11 PM" src="https://user-images.githubusercontent.com/14876567/82098572-0e867e00-96ba-11ea-9853-684ac024538d.png">

**Comment by natalieworth** [comment]
I opened a [dev ticket for this here](https://docs.google.com/document/d/1ZXUXbFX4LdqAYLfzGihOmkQp-gVJzt0p5WxexuaKsCI/edit#), so closing this issue now :)
